src/app/models/submission.server.model.ts

```diff
@@ -115,6 +81,17 @@ const emailSubmissionSchema = new Schema<IEmailSubmissionSchema>({
   },
 })
 
+/**
+ * Returns an object which represents the encrypted submission
+ * which will be posted to the webhook URL.
+ */
+
+emailSubmissionSchema.methods.getWebhookView = function (
+  this: ISubmissionSchema,
```

Comment:
Don't need `this` here ba, since we don't use `this` inside the function

Reply:
will do